flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/plan/stream/sql/join/LookupJoinTest.scala

```diff
@@ -752,7 +752,8 @@ class LookupJoinTest(legacyTableSource: Boolean) extends TableTestBase with Seri
   def testJoinHintWithTableAlias(): Unit = {
     // TODO to be supported in FLINK-28850 (to make LogicalSnapshot Hintable)
     thrown.expectMessage(
-      "The options of following hints cannot match the name of input tables or views")
+      "The options of following hints cannot match the name of input tables or" +
+        " views: \nLOOKUP:D")
```

Comment:
The message looks not clear. at first glance, I thought `Lookup:D` was a table name